tests/conftest.py

```diff
@@ -15,7 +15,7 @@
 except ImportError:
     pytest_plugins = []
 else:
-    pytest_plugins = ['hypothesis.extra.pytestplugin']
+    pytest_plugins = []
```

Comment:
I think this is a mistake?

Reply:
yes, sorry I get a 
```
ValueError: Plugin already registered: hypothesis.extra.pytestplugin=<module 'hypothesis.extra.pytestplugin' from 'envs/pydantic/lib/python3.8/site-packages/hypothesis/extra/pytestplugin.py'>
```
when this is enabled.  But in any case it's a mistake... I have to poke around a bit to try and figure out why that's happening, I'll file an issue if I actually find out what's going on.

Changed it.